@here/harp-mapview/lib/MapView.ts

```diff
@@ -614,6 +614,31 @@ export interface MapViewOptions {
      * @default false.
      */
     synchronousRendering?: boolean;
+
+    /**
+     * Set initial center of the scene in geo coordinates.
+     * Longitude values outside of -180 and +180 are acceptable.
+     * Default is `new GeoCoordinates(25, 0, 30000000)`
+     */
+    geoCenter?: GeoCoordinates;
```

Comment:
Please accept `GeoPointLike | GeoCoordinatesLike | LatLngLike` so people can use simpler forms they're accustomed with.

Reply:
I added this functionality now.